external/storm-pmml/src/main/java/org/apache/storm/pmml/runner/jpmml/JpmmlFactory.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ *   or more contributor license agreements.  See the NOTICE file
+ *   distributed with this work for additional information
+ *   regarding copyright ownership.  The ASF licenses this file
+ *   to you under the Apache License, Version 2.0 (the
+ *   "License"); you may not use this file except in compliance
+ *   with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.storm.pmml.runner.jpmml;
+
+import org.apache.storm.pmml.runner.ModelRunnerFactory;
+import org.dmg.pmml.IOUtil;
+import org.dmg.pmml.PMML;
+import org.jpmml.evaluator.Evaluator;
+import org.jpmml.evaluator.ModelEvaluator;
+import org.jpmml.evaluator.ModelEvaluatorFactory;
+import org.jpmml.manager.PMMLManager;
+import org.xml.sax.SAXException;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Objects;
+
+import javax.xml.bind.JAXBException;
+
+/*
+ * This class consists exclusively of static factory methods that create
+ * object instances that are essential to work with the Jpmml library
+ */
+public class JpmmlFactory {
+    /**
+     * Creates a new {@link PMML} object representing the PMML model defined in the XML {@link File} specified as argument
+     */
+    public static PMML newPmml(File file) throws JAXBException, SAXException, IOException {
```

Comment:
@ptgoetz I created [STORM-2240](https://issues.apache.org/jira/browse/STORM-2240) to address this as follow up